sdk/ai/Azure.AI.AgentServer/tests/AgentFramework.Integration/AgentFramework.Integration.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <PackageReference Include="dotenv.net"/>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing"/>
+    <PackageReference Include="Microsoft.AspNetCore.TestHost"/>
+    <PackageReference Include="Microsoft.NET.Test.Sdk"/>
+    <PackageReference Include="xunit"/>
```

Comment:
Outside of legacy code, tests in the repository should be using NUnit.    We'll need to convert these over.